test/ui/cypress/e2e/environment_helpers.js

```diff
@@ -1,10 +1,25 @@
 export function verifyRequiredEnvVariables() {
-	let deploymentDriverUrl = Cypress.env('DEPLOYMENT_DRIVER_URL')
-  let username = Cypress.env('USERNAME')
-  let password = Cypress.env('PASSWORD')
-  expect(deploymentDriverUrl).not.to.be.undefined
-  // uncomment following when we implement SSO login
-  //expect(username).not.to.be.undefined
-  //expect(password).not.to.be.undefined
-	return { deploymentDriverUrl, username, password }
+
+  if (!Cypress.env('DEPLOYMENT_DRIVER_URL')) {
```

Comment:
This sort of check prevents using environment variable with value of empty string or 0. We should check for `undefined` instead. Applicable to all other env variables check below.

Reply:
@tznamena , we should not use an empty string or 0 as the deployment engine UI's URL to run our automation. Right?

Reply:
Thanks for the info. I didn't know that the `!` operator in javascript has side effects.:-) I will update them to use `expect().not.to.be.undefined`.